ledger/acctonline_expired_test.go

```diff
@@ -626,7 +626,7 @@ func TestOnlineAcctModelScenario(t *testing.T) {
 			})
 			// test same scenario on double ledger
 			t.Run("DoubleLedger", func(t *testing.T) {
-				m := newDoubleLedgerAcctModel(t, protocol.ConsensusFuture, true)
+				m := newDoubleLedgerAcctModel(t, protocol.ConsensusV39, true) // TODO simulate heartbeats
```

Comment:
Is this TODO meant to be done down the line/outside scope of this PR?

Reply:
I think that these tests were failing because suspensions were getting in the way of the test doing what it wanted, which was to test _expiration_ not absenteeism or challenges.  So I think I thought I would have to add heartbeats to keep the nodes online for this test to work, and maybe I did.  But now I don't - maybe it's because the parameters are long enough to avoid suspensions during the test, maybe it's because we put in proposers now. Anyway, this test works on vFuture, and it's testing something other than heartbeats, so I'm going to revert it to its old form.